packages/popup-component/src/components/popup/shared.scss

```diff
@@ -0,0 +1,20 @@
+$blue: #d8f6ff;
+$red: #ff4630;
+$purple: #dac4f5;
+$green: #d6ffb8;
+$yellow: #ffed9c;
+$gray: #2b2b2b;
+$beige: #faf9f5;
+$white: #ffffff;
+$black: #000000;
+$grid-gutter-width: 40px;
+$zindex-fixed: 1030;
+$xs: 576px;
```

Comment:
You have `$grid-breakpoints` which you can use like: `map-get($map: $grid-breakpoints, $key: "xs")`

Reply:
We don’t have bootstrap or bootstrap variables here, for these kind of things I don’t think we should really on bootstrap